sys/include/net/gnrc/ipv6/netif.h

```diff
@@ -114,6 +114,13 @@ extern "C" {
 /**
  * @{
  * @name Flags for the interfaces
+ *
+ * @note The most-significant byte of the these flags is identical to the flags
+ *       in IPv6 router advertisements. See <a
+ *       href="https://tools.ietf.org/html/rfc4861#section-4.2">RFC4861,
+ *        section 4.2</a> and <a
+ *       href="https://tools.ietf.org/html/rfc6275#section-7.1">RFC6275,
```

Comment:
I wonder if at the end a link to just https://tools.ietf.org/html/rfc4389#section-4.1.3.3 would have been enough. But anyway, better than nothing.